src/ncp/ncp_base_mtd.cpp

```diff
@@ -2466,6 +2466,22 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CNTR_ALL_MAC_COUNTERS
     SuccessOrExit(error = mEncoder.WriteUint32(counters->mTxErrCca));
     SuccessOrExit(error = mEncoder.WriteUint32(counters->mTxErrAbort));
     SuccessOrExit(error = mEncoder.WriteUint32(counters->mTxErrBusyChannel));
+    SuccessOrExit(error = mEncoder.WriteUint32(OPENTHREAD_CONFIG_MAC_DEFAULT_MAX_FRAME_RETRIES_DIRECT));
```

Comment:
Suggest either adding a new property for the histogram (mainly since it may be optional) or adding the new feilds in a struct (to allow them to be empty when feature is not enabled and new field be added after them).
Also please update the `spinel.h` definitions/documentation for the property format change.